templates/components/itilobject/fields/global_validation.html.twig

```diff
@@ -70,11 +70,45 @@
                'display': false,
                'disabled': (not canupdate),
                'readonly': (field_options.fields_template.isReadonlyField('_add_validation')),
+               'rand': rand,
             }|merge(required)),
             __('Approval request'),
             field_options
          ) }}
 
+         {# Validation step #}
+         {{ fields.dropdownField(
+            'ValidationStep',
+            '_validationsteps_id',
+            validationsteps_id,
+            __('Validation step'),
+            {
+               'rand': rand,
+               'value': call('ValidationStep::getDefault').id,
+            }|merge(field_options)
+         ) }}
+
+         <script type="text/javascript">
```

Comment:
Probably don't need to use this jQuery logic. Make the script tag a module instead. This will delay the execution until the DOM is loaded.
```suggestion
         <script type="module">
```